server/controllers/admin/cruDashboardController.ts

```diff
@@ -53,7 +53,7 @@ export default class CruDashboardController {
 
       const {
         pageNumber = 1,
-        sortBy = 'expected_arrival',
+        sortBy = status === 'matched' ? 'canonical_arrival_date' : 'expected_arrival',
```

Comment:
Hmm, I got confused by the terms here, but it's quite correct. 
Sadly, we have an `expectedArrival` in `Cas1PlacementRequestDetail` but also have `expectedArrivalDate` in `Cas1SpaceBookingDetail`. They are very different things though.

Reply:
Indeed. The UI now does a decent job of showing the right wording (in this case the column is 'Booked arrival date'), but this is the property name as dictated by the API. AT least this only appears in the query string!